album/SECL-2537.toml

```
@@ -0,0 +1,25 @@
+[album]
+album_id = "64600c4a-8b4f-4719-bd30-3fa77cec552e"
+title = "Prover／Tell me"
+artist = "milet"
+date = 2020-02-17
```

```suggestion
date = 2020.02.19
```

https://www.sonymusic.co.jp/artist/milet_music/discography/SECL-2537

确认一下？